### ISSUE TITLE
Evaluate Open PRs

### ISSUE THREAD

**Issue by leckman** [nbproject/nbproject#380]
Goal: figure out action plans for moving forward (or throwing out) the 13 open PRs

**Comment by leckman** [comment]
#378 README: Fix nb.mit.edu url + minor reformatting
- Can be merged w/o conflict and should be merged

#374 Landing Page and FAQ Page Redesign
- Can be merged w/o conflict and should be merged

#371 Remove email and plaintext password from welcome email
- Can be merged w/o conflict, but change is trivial
- Probably good to have a larger conversation about what a good welcome email might look like 

#386 Add ability to directly add users to ensemble
- Can be merged w/o conflict
- Follow up work on notification pane should be done before merge

#366 Bug fix: navigator sendBeacon issue (323)
- Can be merged w/o conflict
- Should be merged after code review and removing logging statements from PR

#363 Add analytics page for HTML
- Can be merged w/o conflict
- Should be reviewed before merge

#361 Bug fix for #323
- Can be merged w/o conflict
- Looks good but pretty much the same as #366
- Should probably merge this and delete #366

#322 Upgrade JqueryUI CSS as well
- Conflict in INSTALL.md
- Should assess that upgrade actually worked before deciding to merge
- Looks like this might have been resolved by #330

#232: Fix issue #226: Prevent Submission of Empty Comments
- Conflict in content/modules/dev/ui.editorview/js
- Merge is shown against master, not dev
- Discussion suggests not merging - would eliminate purposeful “empty” comments meant to highlight
- Should open a new ticket to change display of empty comments instead of taking this PR and disallowing them.

#231 Grunt Watch Extended
- Builds on PR #229
- Lots of conflicts, PR evaluated against master
- Should probably not be merged without #229 (hopefully that would resolve some of the conflicts)
- Questionable value of merging now
- Comment that it’s “in integration” but PR was never merged?

#229 Grunt watch
- Conflict with Gruntfile.js and package.json, against master
- Lints & concatenates JS files
- Concatenates & minifies CSS files
- Looks like merge was approved, comment that it’s “in integration” but PR was never merged?

**Comment by leckman** [comment]
New ticket to #232 is #381
